### PR TITLE
Fix: Invalid findPropertyFiltre filter

### PR DESCRIPTION
**Pull request by morwoen**
When without was false and table[s[property]] was false it was always returning true, instead of the intended false.

### CODE REVIEW

**Comment by democratic-collaboration** [comment]
[democratic collaboration](https://github.com/TooAngel/democratic-collaboration)
Approved reviews will speed up the merge, request changes will slow it down.

Please review the PR to help.

Summoning some reviewers:
 - @TooAngel: Please look at the code
 - @Nevrdid: A proper review is appreciated
 - @Somotaw: I always like your reviews

File: src/prototype_room_utils.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ Room.prototype.findPropertyFiltre = function(findTarget, property, properties, w
   let table = {};
```
Comment (TooAngel) [inline src/prototype_room_utils.js]: If you touch this, could you also quickly fix the method name to `findPropertyFilter`

Comment (morwoen) [inline src/prototype_room_utils.js]: Sure 👍

**Comment by democratic-collaboration** [comment]
DCBOT: A new review, yeah.

    Votes: 157/323
    Coefficient: 0.486068111455
    Merging in 7 days 15 hours
    Age 0 days 1 hours

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 213/325
    Coefficient: 0.655384615385
    Merging in -2 days 23 hours
    Age 6 days 4 hours

**Pull request merged**